engine/src/main/java/org/terasology/rendering/nui/layouts/CardLayout.java

```diff
@@ -26,20 +26,43 @@
 import java.util.Iterator;
 import java.util.Map;
 
+/**
+ * A layout that allows for a single widget to be displayed among a list of stored widgets
+ */
 public class CardLayout extends CoreLayout<LayoutHint> {
+    /**
+     * The id of the currently displayed widget
+     */
```

Comment:
I disagree, because it doesn't help distinguish the intended audience of the various pieces of text for a minor convenience. But I guess I'll have to live with it.